ldap_auth_provider.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, config, account_handler):
         self.ldap_start_tls = config.start_tls
         self.ldap_base = config.base
         self.ldap_attributes = config.attributes
+        self.ldap_bind_pattern = config.bind_pattern
```

Comment:
if this is only used in simple mode, can you make that clear by putting in an `else` arm for the following `if self.ldap_mode == LDAPMode.SEARCH` condition?